contact_search_form/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
It's a good habit to add the last line empty (trailing line) for avoiding to have 2 line diff when adding a new line at the end.